graphql_compiler/typedefs.py

```diff
@@ -16,9 +16,13 @@
 # Hence, the "unused import" warnings here are false-positives.
 if sys.version_info[:2] >= (3, 8):
     # TypedDict and Literal were only added to typing in Python 3.8
-    from typing import Literal, TypedDict  # noqa  # pylint: disable=unused-import
+    from typing import Literal, TypedDict, Protocol  # noqa  # pylint: disable=unused-import
```

Comment:
Maybe update line 8 to not reference specific imports anymore, since it's not just `Literal` and `TypedDict` anymore?

Reply:
See line 8.

Reply:
Ah, didn't see the 1 before the 8